src/components/teampage_infoblock.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import propTypes from "prop-types";
+import styled from "styled-components";
+import Img from "gatsby-image";
+
+const Centered = styled.h5`
+  text-align: center;
+  margin-top: 10px;
+`;
+
+const Centered2 = styled.h6`
+  text-align: center;
+  margin-top: 10px;
+`;
+
+const Centered3 = styled.h6`
+text-align: center;
+margin-bottom: 30px
+flexGrow: 0;
+flexBasis: auto;
+flexShrink: 1;
+flex-wrap: wrap;
+flexDirection: row;
+color: var(--MHP-purple);
+`;
+
+const colour = {
+  color: "#4831d4",
+};
+
+const TeamPage_InfoBlock = ({
+  name,
+  degree,
+  position,
+  photo,
+  linkedIn,
+  id,
+}) => (
+  <div className="col-sm-6 col-md-3" id={id}>
+    {/* Position heading component */}
+    <Centered> {position} </Centered>
+
+    {/* Image component */}
+    <Img fluid={photo} />
+
+    {/* Name, degree and linkedIn heading */}
+    <Centered> {name} </Centered>
+    <Centered2>{degree}</Centered2>
+
+    {/* LinkedIn, will not display if there isn't one */}
+    {linkedIn !== "" && linkedIn !== null && (
+      <Centered3>
+        <a style={colour} href={linkedIn}>
```

Comment:
Really tiny nit pick, but I've written a link component called `Link`. This will just mean that hopefully all of the linking logic stays the same for the whole site.
It should be as easy as going `import Link from "./link";` and changing `<a style={colour} href={linkedIn}>` to `<LinkComponentName to={linkedIn}>` or something similar.